File: logiweb-service/src/main/java/com/alekseytyan/logiweb/dto/GenericResponse.java
package com.alekseytyan.logiweb.dto;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@AllArgsConstructor
@NoArgsConstructor
@Getter @Setter
public class GenericResponse<T> {
    private T attachedObj;
}

File: telegram-bot-service/src/main/kotlin/com/alekseytyan/telegrambot/dto/GenericResponse.kt
package com.alekseytyan.telegrambot.dto

import lombok.AllArgsConstructor
import lombok.Getter
import lombok.NoArgsConstructor
import lombok.Setter

@AllArgsConstructor
@NoArgsConstructor
@Getter @Setter
class GenericResponse<T> {
    val attachedObj: T? = null
}